### PR TITLE
Add afaan-oromoo footer translations

### LINKED ISSUE #3034: Enable Footer for Afaan Oromoo Page
**Is your feature request related to a problem? Please describe.**
We require the footer to be enabled for Afaan Oromoo with the correct translations and correct links

Please refer to the full spec in WSMEDIA-29 (linked to confluence translations is included) 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves https://github.com/bbc/simorgh/issues/3034

**Overall change:** _Add translations and links to the afaan oromoo service config._

**Code changes:**
_Add translations and links to the afaan oromoo service config._

**Test notes:**
_Just needs tested against AC - no additional impact areas.  See notes here: https://github.com/bbc/simorgh/issues/3034_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/lib/config/services/afaanoromoo.js [hunk 1]
```diff
@@ -129,39 +129,31 @@ const service = {
...
+        text: 'BBC News maaliif amanuu dandeessa',
       },
       {
-        href: 'https://www.bbc.com/terms',
-        text: 'Terms of Use',
+        href: 'https://www.bbc.co.uk/usingthebbc/terms/',
+        text: 'Haala itti fayyadamaa',
       },
       {
-        href: 'https://www.bbc.co.uk/aboutthebbc/',
```
Comment (pjlee11) [inline src/app/lib/config/services/afaanoromoo.js:136]: Are we meaning to lose the `aboutthebbc` link from the footer? It's currently on the production site https://www.bbc.com/afaanoromoo

File: src/app/lib/config/services/afaanoromoo.js [hunk 2]
```diff
@@ -129,39 +129,31 @@ const service = {
...
-        text: 'Privacy Policy',
+        href: 'https://www.bbc.co.uk/usingthebbc/privacy/',
+        text: 'Imaammata mateenyaa',
       },
       {
         href: 'https://www.bbc.com/usingthebbc/cookies/',
-        text: 'Cookies',
-      },
-      {
-        href: 'https://www.bbc.com/accessibility/',
```
Comment (pjlee11) [inline src/app/lib/config/services/afaanoromoo.js:148]: Similarly to `aboutthebbc` is the `accessibility` link being dropped?

File: src/app/lib/config/services/afaanoromoo.js (hunk 1 above)
Comment (ryanmccombe) [reply src/app/lib/config/services/afaanoromoo.js:136]: Not sure - these were the only translations provided

I asked Jane over on the issue: https://github.com/bbc/simorgh/issues/3034

File: src/app/lib/config/services/afaanoromoo.js (hunk 2 above)
Comment (pjlee11) [reply src/app/lib/config/services/afaanoromoo.js:148]: Can you confirm if these links are being removed on purpose? If so I'll approve

Comment (ryanmccombe) [reply src/app/lib/config/services/afaanoromoo.js:148]: Seems to be intentional - @aeroplanejane confirmed at standup.  See notes / Jane's comment here: https://github.com/bbc/simorgh/issues/3034

**Review by pjlee11 — APPROVED** [review]

**Review by 12 — APPROVED** [review]
LGTM

**Comment by staylos92** [comment]
This does not require any additional testing to that covered on https://github.com/bbc/simorgh/issues/3034#issuecomment-520441179

**Pull request merged**